### DISCUSSION TITLE
这东西宝塔咋搭建啊

### DISCUSSION THREAD

**Discussion by 101200** [AlistGo/alist#597] [Q&A]
我用网上找的宝塔教程整不明白啊

**Comment by jing5460** [comment]
作者文档

**Reply by 101200** [reply]
![image](https://user-images.githubusercontent.com/38938307/155208526-69950a55-bf4c-4d33-9d40-0751199740fa.png)
脚本还是手动

**Reply by jing5460** [reply]
都可以

**Comment by hainc** [comment]
> 我用网上找的宝塔教程整不明白啊

https://www.haah.net/archives/7202.html

**Reply by hainc** [reply]
> > 我用网上找的宝塔教程整不明白啊
> 
> https://www.haah.net/archives/7202.html

首页有Q群，加群他们会教你

**Comment by Cathgao** [comment]
先一键安装，等服务安装好后，记下服务端口。
去宝塔里的web服务器（一般是nginx）新建一个站点，然后将站点转发到`http://127.0.0.1:<port>`，port为安装时提示的服务端口

文档里也有对反代进行设置的章节，请先仔细阅读
https://alist-doc.nn.ci/docs/install/reverse-proxy

**Comment by xunzh2000** [comment]
https://masuc.cn/19373.html?ivk_sa=1024320u 参照这个，可以的，我用了。
